ios/FluentUI/Vnext/Drawer/SlideOverPanel.swift

```diff
@@ -64,6 +67,7 @@ struct MSFSlideOverPanel<Content: View>: View {
                         radius: tokens.shadowBlur,
                         x: tokens.shadowDepthX,
                         y: tokens.shadowDepthY)
+                .animationCompletion(value: Double(resolvedContentOffset), completion: transitionCompletion)
```

Comment:
Shouldn't we call these "onAnimationComplete" to follow the event modifiers naming pattern? 
Like "onAppear", "onDisappear", etc...